src/app/create-event/page.tsx

```diff
@@ -62,113 +57,111 @@ const CreateEventPage: NextPage = () => {
     { value: 'Any', label: 'Pro každého' },
   ];
 
-  const watchedValues = watch();
+  const watchedDescription = methods.watch('description');
```

Comment:
Well, there are two options we can do to save using the word `methods` 36 times. The saves bet will to destructure it underneath the hook